.npmignore

```diff
@@ -0,0 +1,2 @@
+/.github
+/test
```

Comment:
We should copy-paste the `.gitignore` file here or the local `.vscode/` directory would be released.

Could you restore the `test/` folder?
It is an org policy to ship it within the npm package.